app/code/Magento/Theme/Model/Wysiwyg/Storage.php

```diff
@@ -296,8 +296,13 @@ public function getFilesCollection()
             if (self::TYPE_IMAGE == $storageType) {
                 $requestParams['file'] = $fileName;
                 $file['thumbnailParams'] = $requestParams;
-                //phpcs:ignore Generic.PHP.NoSilencedErrors
-                $size = @getimagesize($path);
+
+                try {
+                    $size = getimagesizefromstring($this->file->fileGetContents($path));
```

Comment:
What's the reason of replacing this one? The docs show that [`getimagesize`](https://www.php.net/manual/en/function.getimagesize.php) is still supported by PHP8.

Reply:
Hello @eduard13, thank for your review, I changed function getimagesize() to getimagesizefromstring() because getimagesize() is discouraged. (static test failed) .

Thank you